__tests__/Rating/rating.test.ts

```diff
@@ -0,0 +1,283 @@
+// import axios from "axios";
+import { ObjectId } from "bson";
+import {
+	addRating,
+	deleteRatingFromDb,
+	editRatingInDb,
+	getAllRatingsForPlace,
+	getAllRatingsFromUser,
+	getRatingById,
+} from "../../src/db/Rating/rating";
+import { getPlaceByID } from "../../src/db/Place/place";
+import { dbConnection } from "../../src/db/mongoConnection";
+import { Rating, Place } from "../../src/db/types";
+import { addPlace } from "../../src/db/Place/place";
+import { addUserToDb } from "../../src/db/User/user";
+
+jest.setTimeout(10000);
```

Comment:
```suggestion
```

Reply:
The `setTimeout` is needed here because the `beforeAll` takes longer than the default 5 seconds @carpenterd777